curriculum/challenges/english/08-coding-interview-prep/rosetta-code/fizzbuzz.english.md

```diff
@@ -0,0 +1,92 @@
+---
+title: FizzBuzz
+id: 5e9ddb06ec35240f39657419
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+
+Write a program that outputs the integers from 1 to 100 (inclusive). But: <br>
+<ul>
+    <li> for multiples of 3, output <b>Fizz</b> instead of the number </li>
+    <li> for multiples of 5, output <b>Buzz</b> instead of the number </li>
+    <li> For multiples of 3 and 5, output <b>FizzBuzz</b> instead of the number </li>
+</ul>
```

Comment:
Sorry, I'm not sure what happened to my comment here from yesterday.

Let's clean up some of the spacing here, add code blocks and quotation marks around the strings that should be returned, and change the text slightly:

```
Write a program that returns an array of integers from 1 to 100 (inclusive). But:
<ul>
    <li>for multiples of 3, add <code>"Fizz"</code> to the array instead of the number</li>
    <li>for multiples of 5, add <code>"Buzz"</code> to the array instead of the number</li>
    <li>for multiples of 3 and 5, add <code>"FizzBuzz"</code> to the array instead of the number</li>
</ul>
```